src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -0,0 +1,167 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps #pylint: disable=unused-import
+
+#pylint: disable=line-too-long
+
+helps["sf cluster list"] = """
+    type: command
+    short-summary: List all the cluster resource in the same subscription
+"""
+
+helps["sf cluster new"] = """
```

Comment:
okay, i will update, yesterday, i forgot to push the change, you can take a look, i just pushed.